pkg/render/logstorage/linseed/linseed.go

```diff
@@ -253,6 +255,56 @@ func (l *linseed) linseedClusterRoleBinding(namespaces []string) client.Object {
 	return rcomponents.ClusterRoleBinding(ClusterRoleName, ClusterRoleName, ServiceAccountName, namespaces)
 }
 
+func (l *linseed) extraLinseedRBACForCloud() []client.Object {
+	if l.cfg.Tenant == nil {
+		return nil
+	}
+
+	var extraRBAC []client.Object
```

Comment:
Let's just call this "objs" - this isn't "extra" rbac, it's required for this mode of operation.